tests/views/test_auto_security.py

```diff
@@ -117,6 +129,16 @@ class AutoSecurityTests(view_tests_base.ViewTestsBase):
             xsrf_action_id=None),
     }
 
+    def init_testbed_stubs(self):
+        """Initializes the App Engine testbed stubs.
+
+        The base class initializes a Datastore stub, which seems to cause
+        problems when we set another app ID (for the task tests). We don't
+        really need Datastore for this, so just override the base class and
+        stick to the user stub.
+        """
+        self.testbed.init_user_stub()
```

Comment:
just checking, should there be a matching self.testbed.deactivate()?

Reply:
The base class does that (the base class also sets these up, just not quite the way we need for this test).